packages/tldraw/src/translations/ar.json

```diff
@@ -97,5 +97,13 @@
   "page": "صفحة",
   "keyboard.shortcuts": "اختصارات لوحة المفاتيح",
   "search": "بحث",
-  "loading": "{dots}تحميل "
-}
\ No newline at end of file
+  "loading": "{dots}تحميل ",
+  "export.background": "تصدير الخلفية",
+  "transparent": "شـفاف",
+  "auto": "ذاتي",
+  "light": "خفيف",
+  "dark": "غامق",
+  "github": "Github",
+  "twitter": "تويتر",
+  "discord": "دسكورد"
```

Comment:
This three doesn't change no matter what language you use, don't put these in there

Reply:
same for github